### PR TITLE
Tag Parameters.jl v0.7.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [mauro3/Parameters.jl](https://github.com/mauro3/Parameters.jl)
Release: [v0.7.0](https://github.com/mauro3/Parameters.jl/releases/tag/v0.7.0)
Travis: [Travis Build Status](https://travis-ci.org/mauro3/Parameters.jl/builds/210056363)
Diff: [vs v0.6.0](https://github.com/mauro3/Parameters.jl/compare/3226946dea5d5d19878d6a8347bb46425533ce80...3cc69d89eb102a825864c927b9c55ec57e5b5c4b)
`requires` vs v0.6.0: 
```diff
--- 0.6.0/requires
+++ 0.7.0/requires
@@ -1,2 +1,3 @@
 julia 0.5
-DataStructures 0.4.3
+DataStructures 0.5.2
+Compat 0.19.0
```
cc: @mauro3

### CODE REVIEW

File: Parameters/versions/0.7.0/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.5
+DataStructures 0.5.2
+Compat 0.19.0
```
Comment (tkelman) [inline Parameters/versions/0.7.0/requires]: 0.17 for abstract type

**Comment by mauro3** [comment]
Additional comments here https://github.com/mauro3/Parameters.jl/commit/16579e1662663ff8a6b7fb9431139ab9e3046643#commitcomment-21271168

**Comment by mauro3** [comment]
@tkelman thanks for the comments!  I hope I've addressed them correctly.

**Comment by mauro3** [comment]
Anything else I should do here? Thanks!

**Pull request merged**